capten/deployment-worker/internal/activities/plugin_activity.go

```diff
@@ -278,7 +278,7 @@ func (p *PluginActivities) PluginDeployPreActionMTLSActivity(ctx context.Context
 	}
 
 	pluginInitConfigmapName := req.PluginName + pluginConfigmapNameTemplate
-	err = captenSDKClient.CreateCertificates(req.PluginName, req.DefaultNamespace, "capten-issuer", pluginInitConfigmapName, p.k8sClient)
+	err = captenSDKClient.CreateCertificates(req.PluginName, req.DefaultNamespace, "agent-ca-issuer", pluginInitConfigmapName, p.k8sClient)
```

Comment:
agent-cluster-issuer name can be moved to config-map so that hardcoding can be removed.